static_src/util/cf_api.js

```diff
@@ -47,7 +46,9 @@ export default {
     return resources.map((r) => this.formatSplitResponse(r));
   },
 
-  fetch(url, action, multiple, ...params) {
+  fetch(url, _action, multiple, ...params) {
+    // Set a default noop action handler
+    const action = typeof _action === 'function' ? _action : () => {};
```

Comment:
Could this be dangerous because if a dev forgets to pass in an action, they won't know why something isn't working?